l10n_it_ddt/wizard/add_picking_to_ddt.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Nicola Malcontenti Agile Business Group
+#    Copyright (C) Francesco Apruzzese
+#    Copyright (C) 2014 Agile Business Group (http://www.agilebg.com)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+
+from openerp import fields
+from openerp import models
+from openerp import api
+from openerp import _
+from openerp.exceptions import Warning
+
+
+class AddPickingToDdt(models.TransientModel):
+
+    _name = "add.pickings.to.ddt"
+
+    ddt_id = fields.Many2one('stock.ddt')
+
+    @api.multi
+    def add_to_ddt(self):
+        pickings = self.env['stock.picking'].browse(
+            self.env.context['active_ids'])
+        for picking in pickings:
+            if picking.ddt_id:
+                raise Warning(
+                    _("Pickings already in ddt"))
```

Comment:
Pickings -> Picking
Could you also add reference to picking name? ```Picking %s ...```

The same for the other raise in this method